pkg/printers/internalversion/describe.go

```diff
@@ -744,6 +744,8 @@ func describeVolumes(volumes []api.Volume, w PrefixWriter, space string) {
 			printCephFSVolumeSource(volume.VolumeSource.CephFS, w)
 		case volume.VolumeSource.StorageOS != nil:
 			printStorageOSVolumeSource(volume.VolumeSource.StorageOS, w)
+		case volume.VolumeSource.HugePages != nil:
+			printHugePagesVolumeSource(volume.VolumeSource.HugePages, w)
```

Comment:
there is also another func where should place this in. (I'm reviewing on my phone, it's not easy to find that func, sorry)